src/forms/CheckboxField.js

```diff
@@ -20,6 +20,7 @@ const CheckboxField = ({
 }) => (
   <FormField id={input.id} help={help} meta={meta}>
     <Form.Check
+      name={input.name}
```

Comment:
I don't see name in https://react-bootstrap.netlify.com/components/forms/#form-check-props. Am I missing something.

Reply:
It gets passed to the `input` element. https://html.com/tags/input/

Reply:
All of the other field components in bandit-pouch set this already. Just this one missing.